c10/util/typeid.h

```diff
@@ -454,12 +454,12 @@ inline TypeMeta::TypeMeta() noexcept
     : data_(_typeMetaDataInstance<detail::_Uninitialized>()) {
 }
 
-inline constexpr bool operator==(
+inline bool operator==(
```

Comment:
why did we remove it here?

Reply:
because this wasn't truly constexpr. `TypeMeta` objects can't even be constructed in a constexpr way currently. I might find a way to do it in future PRs.